roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1090,7 +1090,7 @@ def set_container_facts_if_unset(facts):
         if 'ovs_image' not in facts['node']:
             facts['node']['ovs_image'] = ovs_image
 
-    if facts['common']['is_containerized']:
+    if facts['common']['is_containerized'] in ['true', 'True']:
```

Comment:
When we don't set `is_containerized` it currently takes on the value of `is_atomic` which would be a boolean value in this section of the code. If set by the user then it will be a string instead so I think we have to treat it like a boolean everywhere we use it.